File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.common.by import By

service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service = service) as driver:
    driver.get("https://www.training-support.net/")
    print("The Title of the page is: ", driver.title)
    driver.find_element(By.ID, "about-link").click()
    print("The title of the new page is: ", driver.title)
    driver.quit()

File: Selenium/Activities/activity1.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;


public class activity1 {
    public static void main(String[] args){
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://www.training-support.net/");
        System.out.println("The title of the page is: " + driver.getTitle());
        driver.findElement(By.id("about-link")).click();
        System.out.println("The title of the new page is: " + driver.getTitle());
        driver.quit();
    }
}
